kuma/notifications/utils.py

```diff
@@ -1,6 +1,11 @@
+import json
+
 from kuma.notifications.models import Watch, Notification, NotificationData
 
 
+browsers: dict = json.loads(open("browsers.json").read())
```

Comment:
@nicolaslara that file is not checked in (and probably shouldn't be). However, this is fails the migration on stage.